docs/hacker-stats-intro/matplotlibrc

```diff
@@ -0,0 +1,5 @@
+axes.spines.left   : True   # display axis spines
```

Comment:
I like the style sheet; let's get this one used across the entire series.

According to the [Matplotlib docs](https://matplotlib.org/3.3.2/tutorials/introductory/customizing.html), we can reference it from a single source of truth using the following pattern:

```python
import matplotlib.pyplot as plt
plt.style.use('./images/presentation.mplstyle')
```

I'd like to propose that we use the style sheet by doing the following:

1. Place the `matplotlibrc` file in the project root directory.
2. Reference it at the top of every notebook (I'll take care of that for the other notebooks I've written), using the following pattern:

```python
import matplotlib.pyplot as plt
from pyprojroot import here
plt.style.use(here() / "matplotlibrc")
```

Reply:
HBA to move `matplotlibrc` into `config/matplotibrc`

and load at top of both NBs.